docs/skopeo.1.md

```diff
@@ -64,6 +64,11 @@ Uses the system's trust policy to validate images, rejects images not trusted by
 
   _destination-image_ use the "image name" format described above
 
+  **--authfile** _path_
+
+  Path of the authentication file. Default is ${XDG_RUNTIME\_DIR}/containers/auth.json, which is set using `kpod login`
```

Comment:
Missing period after kpod login.  A mention of precedence would probably be good here too.  Perhaps: "Default is ${XDG_RUNTIME\_DIR}/containers/auth.json, which is set using `kpod login`.  If the authorization state is not found there, then $HOME/.docker/config.json is checked, which is set using `docker login`."

Reply:
fixed